src/main/java/com/squareup/kotlinpoet/ClassName.kt

```diff
@@ -203,6 +213,18 @@ class ClassName private constructor(
       }
       names += getPackage(this).qualifiedName.toString()
       names.reverse()
+      if (names.size == 2 && names[0] == "java.lang") {
+        when (names[1]) {
```

Comment:
Oh, I see what you mean. Like you could have `toMutable` or something like that perhaps.